control/control_grpc_server.go

```diff
@@ -0,0 +1,159 @@
+/*
+http://www.apache.org/licenses/LICENSE-2.0.txt
+
+
+Copyright 2016 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package control
+
+import (
+	"time"
+
+	"github.com/intelsdi-x/snap/core"
+	"github.com/intelsdi-x/snap/core/serror"
+	"github.com/intelsdi-x/snap/grpc/common"
+	"github.com/intelsdi-x/snap/grpc/controlproxy/rpc"
+	"golang.org/x/net/context"
+)
+
+type ControlGRPCServer struct {
+	control *pluginControl
+}
+
+// --------- Schedulers managesMetrics implementation ----------
+
+func (pc *ControlGRPCServer) GetPluginContentTypes(ctx context.Context, r *rpc.GetPluginContentTypesRequest) (*rpc.GetPluginContentTypesReply, error) {
+	accepted, returned, err := pc.control.GetPluginContentTypes(r.Name, core.PluginType(int(r.PluginType)), int(r.Version))
+	reply := &rpc.GetPluginContentTypesReply{
+		AcceptedTypes: accepted,
+		ReturnedTypes: returned,
+	}
+	if err == nil {
```

Comment:
I think you're right that it's not strictly necessary since it would implicitly be an empty string if not set.